custom_components/foxess_modbus/config_flow.py

```diff
@@ -58,7 +66,8 @@ def __init__(self, config=None) -> None:
             {
                 vol.Required(INVERTER_TYPE, default="TCP"): selector(
                     {"select": {"options": ["TCP", "SERIAL"]}}
-                )
+                ),
+                vol.Required(ENERGY_DASHBOARD, default=True): bool,
```

Comment:
Maybe default to false? Nuking the user's current set-up feels a bit dangerous for something which is enabled by default...

Reply:
But it's so cool to do it automatically... why would anyone not want it?! :laughing:

Reply:
E.g. I've got thing set up slightly differently: I don't have my battery yet so I get nonsense for some of the battery sensors, and don't have that added to the energy dashboard, and I'm still using foxess-ha for some sensors until I can work out how to properly migrate my history over. I also need to remove/readd the integration a fair bit, as config changes etc. It only takes one instance of forgetting to uncheck the checkbox and it's gone and nuked everything...